src/axom/inlet/examples/CMakeLists.txt

```diff
@@ -10,22 +10,31 @@
 # Specify list of examples
 #------------------------------------------------------------------------------
 
+set(inlet_examples
+    nested_structs.cpp )
+
 if (SOL_FOUND)
    set(inlet_examples
```

Comment:
Should this be a `list(APPEND ...)` ?  
Or possibly a `blt_list_append`  (https://llnl-blt.readthedocs.io/en/develop/api/utility.html#blt-list-append)

Reply:
I agree that `blt_list_append` would more clearly express intent here.